index.js

```diff
@@ -148,6 +150,8 @@ function randomCase(input) {
 function convert(input, shouldLetterBeUpperCase) {
   return input.replace(/./g, (str, i) => (shouldLetterBeUpperCase(str, i) ? str.toUpperCase() : str.toLowerCase()));
 }
-if (typeof window === "undefined") {
+mockingcase = mOcKiNgCaSe;
```

Comment:
Always try to use `let` or `const`
```suggestion
const mockingcase = mOcKiNgCaSe;
```